<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExpenseDialog</class>
 <widget class="QDialog" name="ExpenseDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(142, 142, 212);</string>
  </property>
  <widget class="QPushButton" name="Expense_Submit_pushButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>230</y>
     <width>135</width>
     <height>39</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	
	background-color: rgb(209, 197, 255);
	color: rgb(39, 180, 164);
	border: 2px solid #e735ff;
border-radius: 10px;
}</string>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Expense_Amount_lineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>150</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	background-color: rgb(151, 164, 225);
	color: rgb(58, 24, 51);
border: 2px solid #e682ff;
border-radius: 10px;
}</string>
   </property>
   <property name="placeholderText">
    <string>Enter expense amount</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Expense_Type_lineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	background-color: rgb(151, 164, 225);
	color: rgb(58, 24, 51);
border: 2px solid #e682ff;
border-radius: 10px;
}</string>
   </property>
   <property name="placeholderText">
    <string>Enter expense type</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>142</width>
     <height>44</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(230, 130, 255);</string>
   </property>
   <property name="text">
    <string>EXPENSE</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>150</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(230, 130, 255);</string>
   </property>
   <property name="text">
    <string>Expense Type:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(230, 130, 255);</string>
   </property>
   <property name="text">
    <string>Expense:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
